pkg/v3/plugin/ocr3.go

```diff
@@ -208,12 +212,14 @@ func (plugin *ocr3Plugin[RI]) ShouldTransmitAcceptedReport(_ context.Context, _
 	}
 
 	for _, upkeep := range upkeeps {
-		if !plugin.Coordinator.IsTransmissionConfirmed(upkeep) {
-			// if any upkeep in the report does not have confirmation, attempt
-			// again
-			return true, nil
+		for _, coord := range plugin.Coordinators {
+			if !coord.IsTransmissionConfirmed(upkeep) {
+				// if any upkeep in the report does not have confirmation, attempt again
+				// TODO: is this logic correct? We skip other upkeeps in the report and other coordinators?
```

Comment:
This might need to be if all are false. This is going to be delicate I think.

Reply:
sorry @EasterTheBunny i didnt quite follow. do you mean we "return true, nil" only if all upkeeps are transmission confirmed(coord.IsTransmissionConfirmed(ALL upkeepS))?

Reply:
Instead of exiting early on a single coordinator returning false, collect all values and return true if all coordinators return false.